ajax/libs/es7-shim/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "es7-shim",
+  "description": "ECMAScript 7 compatibility shims for legacy JavaScript engines",
+  "homepage": "http://github.com/es-shims/es7-shim/",
```

Comment:
Let's remove `"homepage": "http://github.com/es-shims/es7-shim/",`
Because the homepage can be GitHub repo link. (We show this info in repository already.)
Thank you.